test/jdk/java/awt/event/ComponentEvent/TextAreaTextEventTest.java

```diff
@@ -56,7 +56,6 @@ public static void main(String[] args) throws Exception {
             robot.setAutoWaitForIdle(true);
```

Comment:
If you call `waitForIdle` explicitly, this may be redundant. Yet it may make the test more stable as the robot allows processing all the events before the next event is generated.